Resources/config/intl.xml

```diff
@@ -6,6 +6,7 @@
 
     <parameters>
         <parameter key="sonata.intl.locale_detector.request.class">Sonata\IntlBundle\Locale\RequestDetector</parameter>
+        <parameter key="sonata.intl.locale_detector.request_stack.class">Sonata\IntlBundle\Locale\RequestStackDetector</parameter>
```

Comment:
This is not recommended by SF best practices anymore. Please use class names directly.